data/state/account.go

```diff
@@ -217,7 +217,7 @@ func (a *Account) AddToDeveloperReward(value *big.Int) error {
 	err := a.setDeveloperRewardWithJournal(newDeveloperReward)
 	if err != nil {
 		log.Debug("SetDeveloperRewardWithJournal error", "error", err.Error())
```

Comment:
Do you still need to log it here if the error is returned?

Reply:
deleted